docs/src/modules/components/ThemeContext.js

```diff
@@ -109,11 +108,8 @@ if (process.env.NODE_ENV !== 'production') {
 
 export function ThemeProvider(props) {
   const { children } = props;
-  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-  const pageContextValue = React.useContext(PageContext);
-  // `activePage` does not exist for playground pages
-  // forcing light mode in playground avoids the need for a wrapping theme in playground pages
-  const preferredMode = pageContextValue.activePage && prefersDarkMode ? 'dark' : 'light';
```

Comment:
This is the fix, blog posts don't have active pages, like many other marketing pages.